### DISCUSSION TITLE
What is the purpose of the "Save individual images" function?

### DISCUSSION THREAD

**Discussion by KyuubiYoru** [hlky/stable-diffusion-webui#440] [General]
I only see that the output is shown twice, which looks like a bug.
Did I miss something?

![image](https://user-images.githubusercontent.com/33806787/187930279-d5d78832-460a-49f0-9427-8c6e738eaf84.png)

**Comment by altryne** [comment]
But was fixed, the purpose of this flag is to uncheck if you're doing TONS of experimenting and the individual files that get generated during this is not interesting to you to save
